Parser.rb

```diff
@@ -0,0 +1,14 @@
+module Parser
+
+  def self.read(file_name)
+    flashcards = []
+    File.foreach(file_name) do |card|
+      flashcards << card.chomp
+    end
+    flashcards.delete_if{ |file_string| file_string == "" }
+    flashcards
+  end
+
+end
```

Comment:
I would suggest that you turn this module and its method into a "Flashcard Factory". That is to say, the input should be a filename and the output should be an array of freshly instantiated  Card objects. Then you can call this method and pass its return value directly to your Deck as it gets instantiated rather than having your Deck know how to make Card objects.

Your Deck should know how to manage and manipulate a collection of Card objects, but not necessarily how to create them.